pkg/webservice/dao/queue_info.go

```diff
@@ -33,6 +33,7 @@ type PartitionQueueDAOInfo struct {
 	GuaranteedResource     map[string]int64        `json:"guaranteedResource,omitempty"`
 	AllocatedResource      map[string]int64        `json:"allocatedResource,omitempty"`
 	PreemptingResource     map[string]int64        `json:"preemptingResource,omitempty"`
+	HeadRoom               map[string]int64        `json:"headroom,omitempty"`
```

Comment:
It's not tested whether this value appears properly after a web service call. It's not just this field, but quite a few inside `PartitionQueueDAOInfo`. There's a test case `TestGetPartitionQueuesHandler` which should be improved to see if values from the schedulerContext are mapped back properly. This can be done in a follow-up JIRA.

Reply:
Created https://issues.apache.org/jira/browse/YUNIKORN-2179 so that we can proceed with this one.

Reply:
Good catch @pbacsko !